src/Licensing/ThroughputCollectorHostedService.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Particular.License
+{
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using Particular.License.Contracts;
+
+    class ThroughputCollectorHostedService : IHostedService
+    {
+        public ThroughputCollectorHostedService(ILoggerFactory loggerFactory, PlatformData platformData)
```

Comment:
I am unsure if it makes sense to have this class.
I reckon each collector will have its own polling schedule, and hence this class becomes unnecessary.

Reply:
Well we want one entry point from the host, not 3, hence this class.

Reply:
That can be hidden away in `ThroughputCollectorHostBuilderExtensions.AddThroughputCollector`, thoughts?
Because I reckon we want the collectors themselves registered in DI.

Reply:
I see what you mean though - I'll have a look to see what having 3 will be like